File: src/main/java/ru/market/pricing_service/model/dto/ProductPriceInCheckDto.java
package ru.market.pricing_service.model.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.math.BigDecimal;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ProductPriceInCheckDto {
    private Integer productPriceInCheckId = null;
    private Integer productId;
    private Integer checkId;
    private Integer priceListId;
    private Integer discountId;
    private BigDecimal inputPrice;
    private BigDecimal finalPrice;
    private String priceType;
}

File: src/main/java/ru/market/pricing_service/model/dto/ProductPriceInCheckDto.kt
package ru.market.pricing_service.model.dto

import java.math.BigDecimal

data class ProductPriceInCheckDto(
    val productPriceInCheckId: Int? = null,
    val productId: Int,
    val checkId: Int,
    val priceListId: Int,
    val discountId: Int,
    val inputPrice: BigDecimal,
    val finalPrice: BigDecimal,
    val priceType: String
)
